src/ast/json.rs

```diff
@@ -437,6 +437,18 @@ impl<'arena> Ast<'arena> for Json<'arena> {
     fn is_valid_root(&self, _node_type: Class) -> bool {
         true
     }
+
+    fn debug_name(&self) -> String {
+        match self {
+            Self::True => "True".to_owned(),
+            Self::False => "False".to_owned(),
+            Self::Null => "Null".to_owned(),
+            Self::Str(string) => format!("{}", string),
```

Comment:
This gives empty boxes for strings:

![dag](https://user-images.githubusercontent.com/32932209/107047163-6a9caf80-67bf-11eb-9539-83475f964526.png)

I think we just need to explicitly add quotes `format!("\"{}\"", string)`.